docs/src/components/home/WebsiteBanner.tsx

```diff
@@ -1,10 +1,8 @@
 import * as React from 'react';
 import Box from '@mui/material/Box';
 import Typography from '@mui/material/Typography';
-import Link from 'docs/src/modules/components/Link';
-import ROUTES from 'docs/src/route';
 
-export default function SurveyBanner() {
+export default function WebsiteBanner(props) {
```

Comment:
If we want to make this generic I would change it to:

```
export default function WebsiteBanner(props) {
  return (
    props.content ? <Box
      sx={{
          color: '#fff',
        }}
      >
        {props.content}
      </Typography>
    </Box> : null
  );
```

Reply:
Done!